common.lic

```diff
@@ -102,7 +104,10 @@ module DRC
     result = DRC.bput("rummage /#{parameter} my #{container}", 'but there is nothing in there like that\.', 'looking for .* and see .*', 'While it\'s closed', 'I don\'t know what you are referring to', 'You feel about')
 
     case result
-    when 'but there is nothing in there like that.', 'While it\'s closed', 'I don\'t know what you are referring to', 'You feel about'
+    when 'You feel about'
+      release_invisibility
```

Comment:
It's very defined, on a fresh boot it's going to deadlock hard. Common can't load waiting on common-arcana to load, common-arcana can't load waiting on common to load.